docs/how-to/extending/sql.qmd

```diff
@@ -0,0 +1,186 @@
+# Working with raw SQL strings
+
+While Ibis is all about avoiding raw SQL strings, there are few reasons you
+might need to use them:
+
+- You're porting a large amount of existing code and need to get it working
+  quickly without rewriting it all to ibis expressions
+- You need to access functionality that isn't yet available in a public Ibis
+  API
+- You need to access functionality that is may never exist in Ibis such as
+  extremely backend-specific DDL.
+- Your colleague has decided to give a you a big pile of SQL for which you
+  simply can't even.
```

Comment:
I think levity is fine to an extent, I'm just trying to avoid language that might not translate well/be clear to others that lack the same background. To someone not familiar with that slang this reads like a typo.